src/exchange/group/accounts/group-accounts.controller.js

```diff
@@ -79,17 +92,53 @@ angular.module('Module.exchange.controllers').controller(
       }
     }
 
+    getDefaultDomain() {
+      const name = _.last(this.currentAccount.split('@'))
+      return {
+        displayName: name,
+        name: name
+      };
+    }
+
+    fetchAccountCreationOptions() {
+      this.loadingDomains = true;
+      return this.services.Exchange.fetchingAccountCreationOptions(
+        this.$routerParams.organization,
+        this.$routerParams.productId,
+      )
+        .then((accountCreationOptions) => {
+          const domains = [this.allDomainsOption];
+          _.forEach(accountCreationOptions.availableDomains, domain => {
```

Comment:
You will ended up with an ESLint `arrow-parens` error.
```suggestion
          _.forEach(accountCreationOptions.availableDomains, (domain) => {
```